test/fluree/db/query/sparql_test.cljc

```diff
@@ -254,11 +268,109 @@
               [:bind "?num1" "(* (/ (* (* ?age 4) 3) -2) (/ -4 2))"]
               {"@id" "?person", "person:handle" "?handle"}
               {"@id" "?person", "person:age" "?age"}]
+             where)))
+    (let [query "SELECT ?person ?abs ?bnode ?bound ?ceil ?coalesce ?concat ?contains ?datatype ?day ?encodeForUri ?floor ?hours ?if ?iri ?lang ?langMatches ?lcase ?md5 ?minutes ?month ?now ?rand ?round ?seconds ?sha1 ?sha256 ?sha512 ?str ?strAfter ?strBefore ?strDt ?strEnds ?strLang ?strLen ?strStarts ?struuid ?timezone ?tz ?ucase ?uri ?uuid ?year ?isBlank ?isIri ?isLiteral ?isNumeric ?isUri ?sameTerm
```

Comment:
Would this benefit from a `testing` form w/ a description? Or is it just "test _lots_ of binds"?

Reply:
This is testing that the grammar and translation for all the different SPARQL functions works correctly. I'll add a testing expression for it : )